src/Utilities/constants.js

```diff
@@ -88,11 +89,18 @@ export const rhcdOptions = [
   { label: 'Inactive', value: 'nil' },
 ];
 
-export const updateMethodOptions = [
+const initUpdateMethodOptions = [
   { label: 'yum', value: 'yum' },
   { label: 'dnf', value: 'dnf' },
-  { label: 'rpm-ostree', value: 'rpm-ostree' },
 ];
+export const EdgeParityFilterDeviceEnabled = () => {
+  useFeatureFlag('edgeParity.inventory-list-filter');
```

Comment:
This may not work as intended here, since it is calling a hook outside a component.

It might be more appropriate to call it in `useUpdateMethodFilter` and pass the value on to determine what `updateMethodOptions` should return. Or put the pushing or spreading of `rpm-ostree` directly into the `useUpdateMethodFilter` hook.